modules/core/include/opencv2/core/hal/intrin.hpp

```diff
@@ -757,7 +771,10 @@ namespace CV__SIMD_NAMESPACE {
     OPENCV_HAL_WRAP_BIN_OP_LOGIC(v_int16)
     OPENCV_HAL_WRAP_BIN_OP_LOGIC(v_int32)
     OPENCV_HAL_WRAP_BIN_OP_LOGIC(v_int64)
-
+    OPENCV_HAL_WRAP_BIN_OP_LOGIC_FLT(v_float32)
+    #if CV_SIMD_64F
+    OPENCV_HAL_WRAP_BIN_OP_LOGIC_FLT(v_float64)
+    #endif
```

Comment:
Why do we still trying to add bitwise operations for floating point types?

1. Drop that from intrinsics API.
2. Use `vreinterpret` and/or `v_select` in the code directly with a TODO mark to rework on masks.

Reply:
Fixed.